packages/spec_cli/test/spec_cli_test.dart

```diff
@@ -978,30 +989,71 @@ void main() {
 ''',
         });
 
-        expect(testRenderer!.frames.last, '''
+        final lastFrame = testRenderer!.frames.last;
+        expect(lastFrame, contains('PASS  test/passing_test.dart'));
+        expect(
+          lastFrame,
+          contains('''
  FAIL  test/failing_test.dart
   ✕ failing
     Bad state: fail
     test/failing_test.dart 3:25  main.<fn>
+'''),
+        );
+        expect(
+          lastFrame,
+          contains('''
  FAIL  test/failing_group_test.dart
   group
     ✕ failing
       Bad state: fail
       test/failing_group_test.dart 4:27  main.<fn>.<fn>
- PASS  test/passing_test.dart
+'''),
+        );
 
+        const groupFailingTest = '''
   ● group failing test/failing_group_test.dart:4:5
     Bad state: fail
     test/failing_group_test.dart 4:27  main.<fn>.<fn>
+''';
 
+        const failingTest = '''
   ● failing test/failing_test.dart:3:3
     Bad state: fail
     test/failing_test.dart 3:25  main.<fn>
+''';
 
+        const testResults = '''
 Test Suites: 2 failed, 1 passed, 3 total
 Tests:       2 failed, 3 passed, 5 total
-Time:        00:00:00
-''');
+Time:        00:00:00''';
+
+        if (lastFrame.endsWith('''
+$failingTest
+$testResults
+''')) {
+          expect(
+            lastFrame,
+            endsWith(
+              '''
+$groupFailingTest
+$failingTest
+$testResults
+''',
+            ),
+          );
+        } else {
+          expect(
+            lastFrame,
+            endsWith(
+              '''
+$failingTest
+$groupFailingTest
+$testResults
+''',
+            ),
+          );
+        }
```

Comment:
Is it possible to write this in a nicer way? Is it possible to do some "either or"-matcher or something?

Reply:
https://api.flutter.dev/flutter/package-matcher_matcher/anyOf.html ?

Reply:
I searched so much for this yesterday because I knew it existed, thanks!